config/lava/chromeos/chromeos-flash-template.jinja2

```diff
@@ -0,0 +1,83 @@
+{% extends 'base/kernel-ci-base.jinja2' %}
+{% block main %}
+{{ super () }}
+{% endblock %}
+
+{% block actions %}
+{{ super () }}
+
+actions:
+- deploy:
+    kernel:
+      url: {{ flashing_kernel }}
+    modules:
+      url: {{ flashing_modules }}
+      compression: xz
+    nfsrootfs:
+      url: {{ flashing_rootfs }}
+      compression: xz
+    ramdisk:
+      url: {{ flashing_ramdisk }}
+      compression: gz
+    os: oe
+    timeout:
+      minutes: 15
+    to: tftp
+
+
+- boot:
+    timeout:
+      minutes: 15
+    method: depthcharge
+    commands: nfs
+    prompts:
+      - '/ #'
+
+- test:
+    timeout:
+      minutes: 5
+    docker:
+      image: {{ docker_image }}
+      wait:
+        device: true
+    definitions:
+    - repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: docker-network
+        run:
+          steps:
+            - lava-test-case uname --shell "uname -a"
+            - ping -c 1 172.17.0.1
+            - wget http://172.17.0.1/chromeos/{{ chromeos_image }}
+            - lava-test-case scp --shell "scp -i /keys/id_rsa_chromeos_flash -o ConnectTimeout=3 -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null {{ chromeos_image }} root@$(lava-target-ip):/root"
+      from: inline
+      name: docker-network
+      path: inline/docker-network.yaml
+
+- test:
+    timeout:
+      minutes: 60
+    definitions:
+    - repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: hack
+          description: "hack"
+          os:
+            - debian
+          scope:
+            - functional
+          environment:
+            - lava-test-shell
+        run:
+          steps:
+            - lsblk
+            - ls -l /root
+            - IMAGE_NAME=chromiumos_test_image.bin  /bin/bash /usr/bin/chromeos-flash-modules.sh flash_chromeos enable_rw fixup_tmpfiles
```

Comment:
This is still slightly some work-in-progress but I think the script should be renamed or refactored, to have a clearer distinction between flashing and installing modules.  Also this probably should be installed in `/usr/local` or in `/opt` rather than `/usr/bin`.  We could have `/opt/chromeos/flash`, `/opt/chromeos/install-modules` etc.

Reply:
Also the `scp` command is sending the file using the template variable {{ chromeos_image }} but here the value of `$IMAGE_NAME` is hard-coded.